app/components/mount-row/template.hbs

```diff
@@ -0,0 +1,14 @@
+<td data-title="{{dt.instanceName}}" class="clip">
+  <a href="{{href-to 'container' model.instanceId}}">{{model.instanceName}}</a>
+</td>
+<td data-title="{{dt.path}}" class="clip">
+  {{#copy-inline clipboardText=model.path}}
+    {{model.path}}
+  {{/copy-inline}}
+</td>
+<td data-title="{{dt.permission}}" class="clip">
+  {{model.permission}}
```

Comment:
Add a `displayPermission` property and show a friendly label for the common ones, `ro` and `rw` (`formVolumeRow.opts.ro` and rw).  If something other than those then return the raw value (many odd other things are allowed)